prover/prover_dal/src/fri_witness_generator_dal.rs

```diff
@@ -378,15 +379,24 @@ impl FriWitnessGeneratorDal<'_, '_> {
             sqlx::query!(
                 r#"
                 INSERT INTO
-                    scheduler_dependency_tracker_fri (l1_batch_number, status, created_at, updated_at)
+                    scheduler_witness_jobs_fri (
+                        l1_batch_number,
+                        scheduler_partial_input_blob_url,
+                        protocol_version,
+                        status,
+                        created_at,
+                        updated_at
+                    )
                 VALUES
-                    ($1, 'waiting_for_proofs', NOW(), NOW())
+                    ($1, $2, $3, 'waiting_for_proofs', NOW(), NOW())
                 ON CONFLICT (l1_batch_number) DO
                 UPDATE
                 SET
                     updated_at = NOW()
```

Comment:
similar here - I wonder if we shouldn't update the scheduler_partial_input_blob_url..